.github/workflows/stalebot.yml

```diff
@@ -1,51 +1,52 @@
-          operations-per-run: 140
-          days-before-stale: -1
-          days-before-close: -1
-          days-before-issue-stale: 7
-          days-before-issue-close: 7
-          stale-issue-label: 'status: stale'
-          stale-pr-label: 'status: stale'
-          exempt-issue-labels: 'type: enhancement'
-          only-issue-labels: 'needs: author feedback'
-          close-issue-label: "status: can't reproduce"
-          ascending: true
-      - name: Process Stale Flaky Test Issues
-        uses: actions/stale@v9.0.0
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-          only-issue-labels: 'metric: flaky e2e test'
-          days-before-stale: -1
-          days-before-close: -1
-          days-before-issue-stale: 5
-          days-before-issue-close: 2
-          stale-issue-label: 'status: stale'
-          stale-issue-message: 'This issue is being marked as stale due to inactivity. It will be auto-closed if no further activity occurs within the next 2 days.'
-          close-issue-message: 'Auto-closed due to inactivity. Please re-open if you believe this issue is still valid.'
-          close-issue-reason: 'not_planned'
-          remove-stale-when-updated: true
-          exempt-all-assignees: false
-          enable-statistics: true
-          ascending: true
-          operations-per-run: 120
+    stale:
+        runs-on: ubuntu-20.04
+        permissions:
+            contents: read
+            issues: write
+            pull-requests: write
+        steps:
+            - name: Scan issues
+              uses: actions/stale@v9.0.0
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  stale-issue-message: "As a part of this repository's maintenance, this issue is being marked as stale due to inactivity. Please feel free to comment on it in case we missed something.\n\n###### After 7 days with no activity this issue will be automatically be closed."
+                  close-issue-message: 'This issue was closed because it has been 14 days with no activity.'
+                  operations-per-run: 140
+                  days-before-stale: -1
+                  days-before-close: -1
+                  days-before-issue-stale: 7
+                  days-before-issue-close: 7
+                  stale-issue-label: 'status: stale'
+                  stale-pr-label: 'status: stale'
+                  exempt-issue-labels: 'type: enhancement'
+                  only-issue-labels: 'needs: author feedback'
+                  close-issue-label: "status: can't reproduce"
+                  ascending: true
+            - name: Process Stale Flaky Test Issues
+              uses: actions/stale@v9.0.0
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  only-issue-labels: 'metric: flaky e2e test'
+                  exempt-issue-labels: 'metric: skipped test'
```

Comment:
Prettier adjusted all of the indenting on me, but this is the only line that I added.